plenum/common/member/member.py

```diff
@@ -0,0 +1,24 @@
+from hashlib import sha256
+
+from plenum.common.constants import TXN_TYPE, NYM, TARGET_NYM, TXN_ID, ROLE, VERKEY
+from plenum.common.types import f
+
+
+class Member:
```

Comment:
Is Member a good name here? Probably it's something like 'Actor' or 'TransactionContext', or 'TransactionBuilder'?